ts/client/src/accounts/mangoAccount.ts

```diff
@@ -1380,6 +1380,8 @@ export class PerpPosition {
       throw new Error("PerpPosition doesn't belong to the given market!");
     }
 
+    this.updateSettleLimit(perpMarket);
+
```

Comment:
Mh, this is a side-effect in what looks like a read-only accessor function and it doesn't even change what is returned by it. Why is it good to place it here?